src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -262,7 +262,7 @@ public static Exception LogExceptionMessage(Exception exception)
         public static Exception LogExceptionMessage(EventLevel eventLevel, Exception exception)
         {
             if (exception == null)
-                return null;
+                throw new ArgumentNullException(nameof(exception), "null exception was passed to logger");
```

Comment:
does the message add any value in this case? or can we just pass the param name?

Reply:
@keegan-caruso We could just pass the param name, but the I think the message would help more quickly identify why and where this ArgumentNullException happened.

Reply:
"why" is irrelevant in this case. It is a ANE, it should be self-describing.

"where" should be clear from a stack trace. If we need more we should use an IDX prefaced log message.

Reply:
Fair enough, I will remove the exception message. Thanks Keegan!